src/axom/multimat/docs/sphinx/multimat_field_concepts.rst

```diff
@@ -0,0 +1,118 @@
+.. ## Copyright (c) 2017-2024, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Field Concepts
+******************************************************
+
+Fields are data that are defined over a mesh, typically with one or more values
+per cell. Fields can be scalar, indicating 1-component per cell - or they can
+contain multiple components as with vector data (2+ components per cell). This
+section talks about important MultiMat field concepts that determine where fields
+live on the mesh and how their data are organized in memory.
+
+#######################
+Field Mapping
+#######################
+
+MultiMat is associates materials with cells in a mesh, possibly subdividing cells.
+MultiMat includes the concept of field *mapping*, which is where on the mesh the
+field data live. Fields can be defined over the cells, which is how most simulations
+think about cell-centered fields. With MultiMat, fields can also be defined over
+the materials, allowing for compact storage of material-level data. Fields can
+also be defined over the cells/material pairs from the Cell-Material Relation (CMR),
+allowing fields to have data values for each material in a cell.
+
+.. figure:: figures/mapping.png
+   :figwidth: 700px
+
+   Diagram showing field mapping concept.
+
++--------------------+----------------------------------------------------------+
+| FieldMapping       | Meaning                                                  |
++====================+==========================================================+
+| PER_CELL           | The field contains up to ncells*ncomponents values (for  |
```

Comment:
Minor: A space before and after the `*` in this table and elsewhere may make the sizing dimensions more clear.
(ncells * ncomponents versus ncells*ncomponents).

Reply:
I added spaces.